chapters/ch08/1.signals-boosting.ipynb

```diff
@@ -1134,13 +1176,12 @@
     "signal_weight = 1 #can make this a function to differentiate weights for different signal types\n",
     "\n",
     "time_decay_aggregation = f\"\"\"\n",
-    "SELECT query, doc, sum(time_weighted_boost) AS boost FROM (\n",
-    "  SELECT user, query, doc, {signal_weight} *\n",
-    "  POW(0.5, (DATEDIFF('{target_date}', signal_time) / \n",
-    "            {half_life_days}))\n",
+    "SELECT query, doc, SUM(time_weighted_signals.time_weighted_boost) AS boost FROM (\n",
```

Comment:
This change to rewrite 8.7 (`SUM(time_weighted_signals.time_weighted_boost)...`) is not consistent with what was published in the manuscript. Can this be reverted?

<img width="905" alt="image" src="https://github.com/user-attachments/assets/6d04fddd-ab07-4755-9dde-74d53054d713">

Reply:
Note: if the code doesn't work as published, we should obviously change it. If the original still works, though, let's revert.

Reply:
Alright, we lucked out and this qualifier was unneeded, it has been removed.